test/example_data.dart

```diff
@@ -147,6 +147,17 @@ final User fourthUser  = user(fullName: 'Fourth User', email: 'fourth@example');
 // Streams and subscriptions.
 //
 
+/// A fresh stream ID, from a random but always strictly increasing sequence.
+int _nextStreamId() => (_lastStreamId += 1 + Random().nextInt(100));
+int _lastStreamId = 1000;
```

Comment:
nit:
```suggestion
int _nextStreamId() => (_lastStreamId += 1 + Random().nextInt(10));
int _lastStreamId = 100;
```

That way they're less likely to look the same as a random user ID or message ID; and they're smaller than those, which is realistic.